### PR TITLE
HARP-5557: Support for rendering POI icons on every map theme.

### PR DESCRIPTION
**Pull request by kkostecki**
All open-source themes utilizes maki-icons set created with internal
tool-chain (harp-texture-atlas-tools).

TODO:
- move icon texture name solving code from OmvDecodedTileEmitter to more
generic parts,
- clear m_tileLoaderCache in all classes that inherits from TileDataSource.

Signed-off-by: Krystian Kostecki <ext-krystian.kostecki@here.com>

### CODE REVIEW

File: @here/harp-map-theme/resources/berlin_tilezen_base.json [hunk 1]
```diff
@@ -1182,6 +1189,56 @@
...
+                    "iconMayOverlap": true,
+                    "iconReserveSpace": false,
+                    "color": "#090A0B",
+                    "backgroundColor": "#FFFFFF",
+                    "backgroundOpacity": 0.5
+                },
+                "final": true
+            },
+            {
+                "description": "POIs incompatible to tilezen format - draw debug",
```
Comment (zbigg) [inline @here/harp-map-theme/resources/berlin_tilezen_base.json]: remove all "draw debug" and "no kind" styles.

**Comment by zbigg** [comment]
@heremaps-bot ok to test

File: @here/harp-mapview/lib/poi/PoiManager.ts [hunk 2]
```diff
@@ -498,7 +527,11 @@ export class PoiManager {
 
             // imageTextureName may be undefined if a poiTable is used.
             if (imageTextureName === undefined && poiTableName !== undefined) {
-                imageTextureName = "";
+                imageTextureName = `<lookup:  ${poiTableName}[${poiName}]>`;
```
[the remark this answers is not in the document]
Comment (kkostecki) [reply @here/harp-mapview/lib/poi/PoiManager.ts]: In logs better to see that texture name is "promised" to be achieved from poiTable lookup then to see empty string which says nothing...

File: @here/harp-map-theme/resources/berlin_tilezen_base.json (hunk 1 above)
Comment (kkostecki) [reply @here/harp-map-theme/resources/berlin_tilezen_base.json]: This was actually not my idea, but something what is propagated in every theme, usually in worse form (without correct description, etc..).... So are you sure we don't want it anymore?

File: @here/harp-mapview/lib/poi/PoiManager.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (kkostecki) [reply @here/harp-mapview/lib/poi/PoiManager.ts]: Anyway I have removed everything... to speed up discussion.

**Comment by kkostecki** [comment]
I am leaving berlin_* themes styleName untouched ("tilezen") cause there is separate task for switching styleSetName(s). This means that mapview demo app will not yet allow to properly display berlin themes.

**Pull request merged**